bayshop.3m.php

```diff
@@ -47,6 +47,7 @@ enum Status: string
     case SHIPPED = 'Отправленные';
     case CUSTOMS = 'Растаможить товар';
     case READY = 'Готово к выдаче';
+    case WAITING_FOR_COURIER = 'Ожидает курьера';
```

Comment:
Looks like you forgot about `getWeight` method

Reply:
fixed